content/tutorials/webrtc/basics/en/index.html

```diff
@@ -27,125 +27,127 @@
+<p>The <a href="http://dev.w3.org/2011/webrtc/editor/getusermedia.html" title="W3C Editor's Draft: Media Capture and Streams"><code>MediaStream</code> API</a> represents synchronized streams of media. For example, a stream taken from camera and microphone input has synchronized video and audio tracks. (Don't confuse <code>MediaStreamTrack</code> with the &lt;track&gt; element, which is something <a href="http://www.html5rocks.com/en/tutorials/track/basics/" title="HTML5 Rocks: Getting Started With the Track Element">entirely different</a>.)</p>
 
-<p>Probably the easiest way to understand MediaStream is to look at it in the wild:</p>
+<p>Probably the easiest way to understand the <code>MediaStream</code> API is to look at it in the wild:</p>
 
 <ol>
-  <li>In your browser, open the demo at <a href="https://webrtc.github.io/samples/src/content/getusermedia/gum/" title="getUserMedia demo">webrtc.github.io/samples/src/content/getusermedia/gum</a>.</li>
+  <li>In your browser, navigate to <a href="https://webrtc.github.io/samples/src/content/getusermedia/gum/" title="getUserMedia demo">WebRTC samples <code>getUserMedia</code></a>.</li>
   <li>Open the console.</li>
   <li>Inspect the <code>stream</code> variable, which is in global scope.</li>
 </ol>
 
-<p>Each MediaStream has an input, which might be a MediaStream generated by <code>getUserMedia()</code>, and an output, which might be passed to a video element or an RTCPeerConnection.</p>
+<p>Each <code>MediaStream</code> has an input, which might be a <code>MediaStream</code> generated by <code>getUserMedia()</code>, and an output, which might be passed to a video element or an <code>RTCPeerConnection</code>.</p>
 
-<p>The <code>getUserMedia()</code> method takes a <a href="#toc-constraints" title="Internal link to section about Media Constraints">MediaStreamConstraints object</a> parameter, and returns a Promise that resolves to a MediaStream object.</p>
+<p>The <code>getUserMedia()</code> method takes a <a href="#toc-constraints" title="Internal link to section about Media Constraints"><code>MediaStreamConstraints</code> object</a> parameter and returns a <code>Promise</code> that resolves to a <code>MediaStream</code> object.</p>
 
-<p>Each MediaStream has a <code>label</code>, such as'Xk7EuLhsuHKbnjLWkW4yYGNJJ8ONsgwHBvLQ'. An array of MediaStreamTracks is returned by the <code>getAudioTracks()</code> and <code>getVideoTracks()</code> methods.</p>
+<p>Each <code>MediaStream</code> has a <code>label</code>, such as <code>'Xk7EuLhsuHKbnjLWkW4yYGNJJ8ONsgwHBvLQ'</code>. An array of <code>MediaStreamTracks</code> is returned by the <code>getAudioTracks()</code> and <code>getVideoTracks()</code> methods.</p>
```

Comment:
Nit: should be `<code>MediaStreamTrack</code>s`.

Reply:
I'll adjust